packages/yoastseo/src/languageProcessing/languages/fr/helpers/checkIfWordIsComplex.js

```diff
@@ -45,16 +46,19 @@ export default function checkIfWordIsComplex( config, word ) {
 	if ( word[ 0 ].toLowerCase() === word[ 0 ] ) {
 		/*
 		 * If a word is longer than 9 characters and doesn't start with capital letter,
-		 * we check further whether it is a plural ending in -s. If it is, we remove the -s suffix
+		 * we check further whether it is a plural that ends on the -s or -x plural suffix. If it is, we remove the plural suffix
 		 * and check if the singular word can be found in the frequency list.
+		 * if it is a plural that does not end on -s or -x but on -aux, we replace the plural -aux suffix with the singular suffix -al.
 		 * The word is not complex if the singular form is in the list.
 		 */
-		if ( word.endsWith( "s" ) ) {
+		if ( word.test( morphologyData.suffixGroupsComplexity.standardSuffixesWithSplural ||
+			morphologyData.suffixGroupsComplexity.standardSuffixesWithXplural ) ) {
 			word = word.substring( 0, word.length - 1 );
-			return ! frequencyList.includes( word );
+		} else if ( word.endsWith( morphologyData.suffixGroupsComplexity.irregularPluralSingularSuffixes ) ) {
```

Comment:
Since `irregularPluralSingularSuffixes` is an array of two elements: `[ "aux", "al" ]`, and here we want to check if a `word` ends with `"aux"`, then the check should be adjusted like so:
```
 word.endsWith( morphologyData.suffixGroupsComplexity.irregularPluralSingularSuffixes[ 0 ] )
```

`"aux"` is the first element in the array, hence you'd want to access the `0` index of the array.